pkg/apis/garden/helper/helpers.go

```diff
@@ -97,3 +99,26 @@ func DetermineCloudProviderInShoot(cloudObj garden.Cloud) (garden.CloudProvider,
 	}
 	return cloud, nil
 }
+
+// DetermineLatestMachineImageVersions determines the latest versions (semVer) of the given machine images from a slice of machine images
+func DetermineLatestMachineImageVersions(images []garden.MachineImage) (map[string]garden.MachineImageVersion, error) {
+	var (
+		resultMapVersions = make(map[string]garden.MachineImageVersion)
```

Comment:
```suggestion
		resultMapVersions := make(map[string]garden.MachineImageVersion)
```

`var` block not needed.